package.json

```diff
@@ -18,8 +18,7 @@
     "core-js": "^3.1.4",
     "http-status-codes": "^2.1.4",
     "nuxt": "^2.15.3",
-    "sbc-common-components": "^2.4.26",
-    "typescript": "^4.3.2"
```

Comment:
Typescript is only needed to build the project. After that it's all native Javascript.